Tree/distance_between_two_node_of_a_BT.cpp

```diff
@@ -0,0 +1,97 @@
+/*distance between two nodes of a binary tree
+ the distance between two nodes is the minimum number of edges to be traversed to reach one node from another
+ALGORITHM
+1.find the LCA(lowest common ancestor)
+2.find the distance of n1(d1) & n2(d2) from LCA
+3.return (d1 + d2)
+*/
+
+#include<iostream>
+
+using namespace std;
+
+struct Node{
+    int data;
+    Node* left;
+    Node* right;
+
+    Node(int val){
+        data=val;
+        left=NULL;
+        right=NULL;
+    }
+
+};
+ 
+Node* LCA(Node* root, int n1,int n2){
+    if(root==NULL){
+        return NULL;
+    }
+
+    if(root->data == n1 || root->data == n2){
+        return root;
+    }
+
+    Node* left=LCA(root->left,n1,n2);
+    Node* right=LCA(root->right,n1,n2);
+
+    if(left!=NULL && right!=NULL){
```

Comment:
Add Space in-between " != "
and do this everwhere

Reply:
done.